eng/icu.mk

```diff
@@ -91,8 +91,8 @@ data-$(2): $(TARGET_OBJDIR)/$(1)/.stamp-configure | $(TARGET_OBJDIR)/$(1) $(TARG
 
 endef
 
-ifeq ($(TARGET_OS),browser)
-$(eval $(call TargetBuildTemplate,icudt_browser,icudt))
```

Comment:
Does this change the name of the binary?  If so, you should look in runtime to see if it hardcodes the name anywhere.

Reply:
No, this changes icudt_browser.json file from which we create final icudt.dat file and in runtime we use icudt.dat.